components/FullEventCard/FullEventCard.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { StyleSheet, View, Button, Text, ScrollView, CheckBox, TouchableOpacity } from 'react-native';
+
+export default function FullEventCard() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.closeButton}><Button title='X'/></View>
+      <Text style={styles.titleText}>Volleyball</Text>
+      <Text style={styles.text}>Location: Wash Park</Text>
+      <Text style={styles.text}>Start Time: 2:00PM</Text>
```

Comment:
I like incorporating the time onto the cards. We could also implement that onto the pins when a user clicks on those on the map